.github/workflows/pr_test.yml

```diff
@@ -82,3 +82,18 @@ jobs:
       env:
         TEST_COMPUTE_INIT: true
         TEST_COMPUTE_BUILD: true
+  config:
```

Comment:
Move to step before: https://github.com/fastly/cli/blob/06b2ff56e365f98086d049408e5c89206825cae3/.github/workflows/tag_release.yml#L21 to prevent user having to run `make config`.